features/step_definitions/deployment.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require 'active_support/core_ext/hash/slice.rb'
-require 'active_support/core_ext/hash/transform_values.rb'
```

Comment:
I know this is causing deprecation warning. But we still support running on 2.4. Perhaps require it conditionally if RUBY version is above 2.5?

Reply:
or shall we stop supporting ruby 2.4? Maybe ask @jianlinliu

Reply:
@jianlinliu WDYT about the stop support for ruby 2.4?

Reply:
@jianlinliu can you please comment on this?

Reply:
I am not sure what is meaning of support for ruby 2.4. I have no any cucushift test cases.

Reply:
@akostadinov @jianlinliu with F32 out now, the default ruby is 2.7.1, can we merge this PR?  Otherwise, the cucumber output will have lots of warnings.  @jianlinliu what it means is that we would update our jenkins slave pods to have ruby > 2.5